src/Data/Nat/Base.agda

```diff
@@ -183,6 +192,12 @@ data _≤′_ (m : ℕ) : ℕ → Set where
 _<′_ : Rel ℕ 0ℓ
 m <′ n = suc m ≤′ n
 
+------------------------------------------------------------------------
+-- Smart constructors of _<′_
+
+pattern n<′1+n           = ≤′-refl
```

Comment:
I welcome suggestions... there's a tension between the constructor name (modified-by-primes) vs. the name of the corresponding property onto which this constructor will map in the equivalence proof...

Reply:
I think I'd probably prefer to maintain consistency with the former rather than the latter. Logically the former are more likely to be used together, right?

Reply:
I've gone for `<′-base`.